<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XRClient</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="url_lbl">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Server URL</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="url_ledit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="call_btn">
      <property name="text">
       <string>Call</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="main_tab">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <widget class="QWidget" name="sum_diff_tab">
       <attribute name="title">
        <string>Sum and Difference</string>
       </attribute>
       <widget class="QLabel" name="x_lbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>x:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="y_lbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>y:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="x_ledit">
        <property name="geometry">
         <rect>
          <x>48</x>
          <y>39</y>
          <width>125</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="y_ledit">
        <property name="geometry">
         <rect>
          <x>48</x>
          <y>89</y>
          <width>125</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="diff_lcd">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>90</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="sum_lcd">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>40</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="diff_lbl">
        <property name="geometry">
         <rect>
          <x>227</x>
          <y>90</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Difference (x-y):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="sum_lbl">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Sum (x+y):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="validator_tab">
       <attribute name="title">
        <string>Validator Client</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2" colspan="2">
         <widget class="QProgressBar" name="progress_bar"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passed_ledit">
          <property name="text">
           <string>Passed</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="failed_ledit">
          <property name="text">
           <string>Failed</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QTextEdit" name="pass_tedit"/>
        </item>
        <item row="2" column="3">
         <widget class="QTextEdit" name="failed_tedit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="progress_lbl">
          <property name="text">
           <string>Progress</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
